docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -79,6 +79,9 @@ gateway-error
   This policy is similar to the *5xx* policy but will only retry requests that result in a 502, 503,
   or 504.
 
+reset
+  Envoy will attempt a retry if the upstream server does not respond due to a reset.
```

Comment:
The wording here is a bit off I think: the upstream might have started responding but trigger before it finishes (for example, the upstream server could crash mid-response). Maybe reuse the wording from the 5xx retry policy?

Reply:
@snowp Which part of the wording for 5xx exactly? This doesn't trigger if any 5xx response code comes back, right?

Reply:
i mean the language about resets, the `does not respond at all (disconnect/reset/read timeout)` bit

Reply:
Ahh, gotcha. Updated.